libimage/manifests/manifests_test.go

```diff
@@ -341,4 +342,12 @@ func TestPush(t *testing.T) {
 	options.ForceCompressionFormat = true
 	_, _, err = list.Push(ctx, destRef, options)
 	assert.NoError(t, err, "list.Push(with ForceCompressionFormat: true)")
+
+	options.MaxRetries = 5
+	_, _, err = list.Push(ctx, destRef, options)
+	assert.NoError(t, err, "list.Push(with MaxRetries)")
+
+	options.RetryDelay = 3 * time.Second
+	_, _, err = list.Push(ctx, destRef, options)
+	assert.NoError(t, err, "list.Push(with RetryDelay)")
```

Comment:
Is this expected to be testing the new fields in `options` in combination with the other settings that were already set above?  Is it verifying that the new options cause any changes in behavior?

Reply:
@nalind Yes this is only testing field, I expect to test actual retry happing on `podman` side where it throws warning on every failed retry attempt.